package.json

```diff
@@ -6,9 +6,12 @@
     "@testing-library/jest-dom": "^5.11.0",
     "@testing-library/react": "^11.2.2",
     "@testing-library/user-event": "^12.0.11",
+    "normalize.css": "^8.0.1",
```

Comment:
Nice PR. Your code looks very clean. The only thing I noticed is that `normalize.css` doesn't seem to be used anywhere!? 

Fyi: Alternatively you can also use a version for styled-components: https://github.com/sergeysova/styled-normalize

Reply:
Thanks for pointing this out. The styled-component version looks like a better option here. Gonna fix this right away :)